app/views/admin/settings.njk

```diff
@@ -12,7 +14,7 @@
       Turn on start of new cycle
```

Comment:
The title needs to change based on whether the new cycle is turned on or off. You already do this for the button, but not the title.